src/screens/SUE/SueMapViewScreen.tsx

```diff
@@ -1,11 +1,12 @@
 import { StackNavigationProp } from '@react-navigation/stack';
 import * as Location from 'expo-location';
-import React, { useEffect, useState } from 'react';
+import React, { useState } from 'react';
 import { Alert, StyleSheet } from 'react-native';
 
-import { Map, locationServiceEnabledAlert } from '../../components';
+import { Map, SELECTED_MARKER_ID, locationServiceEnabledAlert } from '../../components';
```

Comment:
i need to think about importing the constant from components...hm..it is not a component...same as the alert. we have a bad structure here.
maybe you can import it explicitly from the the file where it is exported from? the same for `locationServiceEnabledAlert`

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/commit/d81789c4260af837113bb002f8dbd1334b778ffd